scripts/release.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+import { argv, cwd } from 'process'
+
+await $`set -e`
+
+const DIR = __dirname
+
+await $`zx ${DIR}/grabthar-validate-git`
+await $`zx ${DIR}/grabthar-validate-npm`
+
+let DIST_TAG = argv.find(element => element.includes('DIST_TAG='))?.replace('DIST_TAG=', '')
```

Comment:
`minimist` is included in zx and that will provide a nice argv parsing tool. let's try to use minimist here